cmd/util/util.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+  "fmt"
+  "os"
+  "os/exec"
+  "strings"
+
+  "github.com/fatih/color"
+)
+
+func CommandOutputContains(cmd []string, value string) bool {
+  return strings.Contains(GetCommandOutput(cmd), value)
```

Comment:
I like keeping it analogous to `GetCommandOutput`. My first thought would actually be making it `DoesCommandOuputContain` as we could also have `DoesCommandOutputEqual`. To me, those names also make it more clear that it returns a boolean.